tests/storage/imageupload.go

```diff
@@ -255,6 +255,53 @@ var _ = SIGDescribe("[Serial]ImageUpload", Serial, func() {
 		)
 	})
 
+	Context("Create upload volume using volume-mode flag", func() {
+		DescribeTable("Should succeed", func(volumeMode string) {
+			sc, exists := libstorage.GetRWOBlockStorageClass()
+			if !exists {
+				Skip("Skip test when RWOBlock storage class is not present")
+			}
+			targetName := "alpine-dv-" + rand.String(12)
+			defer deleteDataVolume(targetName)
```

Comment:
Is this necessary? Isn't the datavolume automatically removed by the test?

Reply:
hmmm it's been some time since I ran these tests but I may have added this because it's in all the tests in this file. I'll check.

Reply:
I'm thinking it's just good practice to manually delete what we are creating before cleaning the namespace, I see it's pretty common to delete the datavolume in our tests.

Reply:
I think it's mostly a past remnant, especially doing it via a defer which will eliminate the DV before it can make it to the debugging artifacts. If ensuring the DV is gone is part of the test, you can do so in the AfterEach

The main motivation to not wait for resource cleanup is to speed up parallel testing. There is cleanup code at the end of every test that calls delete for you

Reply:
Thanks for the clarification @akalenyu. I also tested this without `deleteDataVolume` and works as expected. I'll remove that line and consider opening another PR later to update the whole file. We are not testing DV deletion so I think removing all the delete functions would be fine.

Reply:
Or we can merge this first and then do the whole cleanup in a later PR.